_posts/news/2017-09-01-announcing-scalajs-0.6.20.md

```diff
@@ -60,6 +60,9 @@ scalacOptions += "-P:scalajs:sjsDefinedByDefault"
 {% endhighlight %}
 
 and remove the `@ScalaJSDefined` annotations everywhere in your codebase.
+
+This will prepare your migration for [Scala.js 1.X](http://www.scala-js.org/news/2017/07/03/announcing-scalajs-1.0.0-M1/#non-native-js-classes-by-default-previously-known-as-scalajs-defined).
```

Comment:
Could you write 1.x instead of 1.X, please? That would be more consistent across all our communications.